pkg/asset/machines/machineconfig/authorizedkeys.go

```diff
@@ -0,0 +1,40 @@
+package machineconfig
+
+import (
+	"fmt"
+
+	ignv2_2types "github.com/coreos/ignition/config/v2_2/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ForAuthorizedKeys creates the MachineConfig to set the authorized key for `core` user.
+func ForAuthorizedKeys(key string, role string) *mcfgv1.MachineConfig {
+	if len(key) == 0 {
+		return nil
+	}
+	return &mcfgv1.MachineConfig{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "machineconfiguration.openshift.io/v1",
```

Comment:
`mcfgv1.SchemeGroupVersion.String()` like we do [here][1]?

[1]: https://github.com/openshift/installer/blob/b3b8a8928b1953c00944fae307c933dbcad6cf75/pkg/asset/manifests/dns.go#L60

Reply:
done 362106c1f909e6584ef8a639d127e24a32534700 -> 8967ccd066030ebbb18fb3d19f7dd693df8abc46